x/blob/types/payforblob_test.go

```diff
@@ -172,7 +172,7 @@ func TestValidateBasic(t *testing.T) {
 		{
 			name:    "parity shares namespace id",
 			msg:     paritySharesMsg,
-			wantErr: ErrParitySharesNamespace,
+			wantErr: ErrInvalidNamespace,
```

Comment:
Do we have a test still to check that `ErrPartityShareNamespace` is returned when expected?

Reply:
It should no longer be returned https://github.com/celestiaorg/celestia-app/pull/1590#discussion_r1165765160